# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/AnimationSurfaceProvider.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe image/AnimationSurfaceProvider.cpp
--- a/image/AnimationSurfaceProvider.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/image/AnimationSurfaceProvider.cpp	Wed May 22 22:27:37 2019 +0000
@@ -5,7 +5,7 @@
 
 #include "AnimationSurfaceProvider.h"
 
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/gfx/gfxVars.h"
 #include "nsProxyRelease.h"
 
@@ -464,7 +464,7 @@
   MutexAutoLock lock(mDecodingMutex);
   MOZ_ASSERT(mDecoder);
 
-  return mDecoder->ShouldSyncDecode(gfxPrefs::ImageMemDecodeBytesAtATime());
+  return mDecoder->ShouldSyncDecode(StaticPrefs::ImageMemDecodeBytesAtATime());
 }
 
 RawAccessFrameRef AnimationSurfaceProvider::RecycleFrame(